archlinuxcn/waydroid-git/PKGBUILD

```
@@ -29,5 +29,6 @@ pkgver() {
 }
 
 package() {
+	sed -i -s "s/env python/python/g" "${_pkgname}/waydroid.py"
 	DESTDIR="${pkgdir}" USE_NFTABLES=1 make -C "${_pkgname}" install
 }
```

```suggestion
prepare() {
        sed -i '1s/env //' "${_pkgname}/waydroid.py"
}

package() {
	DESTDIR="${pkgdir}" USE_NFTABLES=1 make -C "${_pkgname}" install
}
```

It it unnecessary to use a global substitution.